src/components/profile/ProfileContent.tsx

```
@@ -268,14 +269,17 @@ export default function ProfileContent({ isMy, userId, sellerDescription, linkUr
 
   const handleConfirmDelete = async () => {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)